lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -144,13 +56,23 @@ Function SetPrivilegeTokens() {
 
 
     if ($myWindowsPrincipal.IsInRole($adminRole)) {
-
+        # Need to adjust token privs when executing Set-ACL in certain cases.
+        # e.g. d:\testdir is owned by group in which current user is not a member and no perms are inherited from d:\
+        # This also sets us up for setting the owner as a feature.
         # See the following for details of each privilege
         # https://msdn.microsoft.com/en-us/library/windows/desktop/bb530716(v=vs.85).aspx
-
-        [void][Ansible.TokenManipulator]::AddPrivilege("SeRestorePrivilege") #Grants all write access control to any file, regardless of ACL.
-        [void][Ansible.TokenManipulator]::AddPrivilege("SeBackupPrivilege") #Grants all read access control to any file, regardless of ACL.
-        [void][Ansible.TokenManipulator]::AddPrivilege("SeTakeOwnershipPrivilege") #Grants ability to take owernship of an object w/out being granted discretionary access
+        Import-PrivilegeUtil
+        $privileges = @(
+            "SeRestorePrivilege",  # Grants all write access control to any file, regardless of ACL.
+            "SeBackupPrivilege",  # Grants all read access control to any file, regardless of ACL.
+            "SeTakeOwnershipPrivilege"  # Grants ability to take owernship of an object w/out being granted discretionary access
+        )
+        foreach ($privilege in $privileges) {
+            $state = Get-AnsiblePrivilege -Name $privilege
+            if ($state -eq $false) {
```

Comment:
It's more for cases where we run these modules as a non admin (who probably won't have these privileges). This probably won't happen in win_acl but I wanted to have this as a try and do, fail later if Windows determines the privilege was actually needed.

We really should be enabling the privileges if the task calls for it rather than doing it in all cases but I didn't want to spend further time changing win_acl for now.